server/index.js

```diff
@@ -0,0 +1,149 @@
+const express = require( 'express' ); // remember to install your npm packages
+const helmet = require( 'helmet' );
+const cors = require( 'cors' );
+const mongoose = require( 'mongoose' );
+const bodyparser = require( 'body-parser' );
```

Comment:
body-parser no longer needed on express servers. Consider the use of express.json() instead.